source/documentation/tools/index.md

```diff
@@ -1,50 +1,39 @@
+* [pydbtools](https://github.com/moj-analytical-services/pydbtools)
+* [mojap-arrow-pd-parser](https://github.com/moj-analytical-services/mojap-arrow-pd-parser)
+* [mojap-metadata](https://github.com/moj-analytical-services/mojap-metadata)
+* [dataengineeringutils3](https://github.com/moj-analytical-services/dataengineeringutils3)
 
-### RShiny
+### R Packages
 
-To use conda in RShiny applications, you will need to use a different `Dockerfile` to deploy the app. The [conda branch of the rshiny-template](https://github.com/moj-analytical-services/rshiny-template/tree/conda) has an appropriate `Dockerfile` for this purpose. This is also necessary if you wish to use Python in your Shiny application, including using `dbtools` for accessing Amazon Athena databases.
+The Analytical Platform community maintain the following R packages:
 
-### Analytical Platform limitations
-
-There are a number of limitations and pitfalls to conda management to be aware of.
-
-## R package versions on Conda
-
-While Anaconda hosts most of the R packages available on CRAN (the Comprehensive R Archive Network), some R packages on Anaconda only have binaries built for certain versions of R. You can identify the available versions by inspecting the first few characters of the Build part of the filename on its page on anaconda.org, like so:
-
-![](images/conda/anaconda_R_version_number_example.PNG)
-
-Alternatively, if you use `conda search PACKAGENAME`, you can look in the Field column:
-
-![](images/conda/conda_search_R_version_number_example.PNG)
-
-If there isn't an appropriate build for a package, attempting to `conda install` that package will result in conda attempting to match the environment to the superior (or inferior) version of R, asking if you want to install/upgrade/downgrade a long list of packages in the process.
-
-Instead, you should install the package locally via `install.packages()` or `remotes::install_github()`. For RShiny apps, you can add an `install.packages()` step to the `Dockerfile` to install additional packages not covered by the conda environment.yml, like so:
-
-```bash
-RUN R -e "install.packages('waffle', repos = 'https://cinc.rud.is')"
-```
+* [dbtools](https://github.com/moj-analytical-services/dbtools)
+* [Rdbtools](https://github.com/moj-analytical-services/Rdbtools)
+* [botor]
```

Comment:
Is botor one of ours? I thought it was made elsewhere

Reply:
```suggestion
```

Reply:
You're right!